src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: $white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__product-name,
+  &__price-value {
+    @include font-weight-500;
+
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    color: $dark-blue;
+  }
+
+  &__price-value,
+  &__buy {
```

Comment:
The `&__price-value` selector is repeated in both the block starting at line 19 and the block starting at line 29. Consider merging these styles to avoid redundancy.